### DISCUSSION TITLE
Adicionar estilo no POUI

### DISCUSSION THREAD

**Discussion by jpinto10** [po-ui/po-angular#1923] [Tire suas dúvidas]
Como adicionar estilo no POUI, na pagina inteira....
IGUAL AO HTML comum!

**Comment by anliben** [comment]
Boa Tarde @jpinto10, tudo bem?

Não consegui entender sua pergunta, pode ser mais específico? Caso esteja falando em, por exemplo, todos os botoes seguirem o estilo poui, isso não é possível

Attr.
